packages/common/client-utils/package.json

```diff
@@ -11,13 +11,26 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
-	"main": "dist/index.cjs",
-	"module": "lib/index.mjs",
+	"exports": {
+		".": {
+			"import": {
+				"default": "./lib/indexNode.mjs",
+				"types": "./lib/indexNode.d.ts"
+			},
+			"require": {
+				"types": "./dist/indexNode.d.ts",
+				"default": "./dist/indexNode.cjs"
+			}
```

Comment:
I don't understand why are-the-types-wrong says that the correct order for "types" and "default" is different in the `import` and `require` cases. I just listened to it.

Reply:
Huh, but putting "default" first causes actual build issues in many places:

```
@fluid-internal/devtools-browser-extension: ERROR in ../devtools-core/lib/data-visualization/DataVisualization.js 16:0-65
@fluid-internal/devtools-browser-extension: Module not found: Error: Default condition should be last one
```

What was wrong was the fact that the types declaration file under lib is a `.d.mts`, not a `.d.ts`, that's the reason it wasn't getting found with `"types": "./lib/indexNode.d.mts",` and the Typescript algorithm was falling back to using the `"default": "./lib/indexNode.mjs"` entry. After fixing the extension for the "types" bit (and adding a mapping to it in the "browser" field just to be safe), things seem to work as expected.